src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -0,0 +1,182 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Http.Logging
+{
+    internal static class LogHelper
+    {
+        private static readonly LogDefineOptions s_skipEnabledCheckLogDefineOptions = new LogDefineOptions() { SkipEnabledCheck = true };
+        private static readonly bool s_logQueryString = GetLogQueryStringSettingValue();
+
+        private static class EventIds
+        {
+            public static readonly EventId RequestStart = new EventId(100, "RequestStart");
+            public static readonly EventId RequestEnd = new EventId(101, "RequestEnd");
+
+            public static readonly EventId RequestHeader = new EventId(102, "RequestHeader");
+            public static readonly EventId ResponseHeader = new EventId(103, "ResponseHeader");
+
+            public static readonly EventId PipelineStart = new EventId(100, "RequestPipelineStart");
+            public static readonly EventId PipelineEnd = new EventId(101, "RequestPipelineEnd");
+
+            public static readonly EventId RequestPipelineRequestHeader = new EventId(102, "RequestPipelineRequestHeader");
```

Comment:
Is it intentional that `RequestHeader` and `RequestPipelineRequestHeader` only differ in their names?

Reply:
>  they are used strictly for different loggers/categories

Does this mean, that it is unlikely that it will confuse log consumer (sink) logic in practice, despite `EventId.Equals()` and `==` only comparing the `Id` values?